package.json

```diff
@@ -3,8 +3,21 @@
   "version": "1.1.0",
   "description": "Extremely fast utf8 only stream implementation",
   "main": "index.js",
+  "type": "commonjs",
+  "types": "types/index.d.ts",
+  "files": [
```

Comment:
Can you remove this?

Reply:
Should I replace it with npmignore then to avoid publishing noise?

Reply:
@mcollina Done.